custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -129,6 +136,19 @@ def patient_type_choice(self):
         }[self.Ptype]
 
     @property
+    def occupation(self):
+        return {
+            '4': 'engineer',
+            '5': 'doctor_',
```

Comment:
Hmm, it looks like that list is out of date now... sorry! 

The list being used currently is here: https://india.commcarehq.org/a/enikshay-test/fixtures/view_lookup_tables/?table_id=da824beb2e4d491377ce135d61b117bf

Not sure if it is easy to map between the list of Nikshay occupations from here: https://docs.google.com/spreadsheets/d/1l29YwNJr1GtP7MPWkIgboFo_Rimcy6ps0Zk6Qt80u5c/edit#gid=248852058
Otherwise, I can recreate that table.